packages/firestore/src/api/database.ts

```diff
@@ -387,7 +438,9 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
       if (this._firestoreClient) {
         return this._firestoreClient.shutdown();
       }
-    }
+    },
```

Comment:
I'm not sure if this should be in INTERNAL. My reasoning was that it seems that in TypeScript, current settings aren't exposed (or I just missed it).

Reply:
INTERNAL is semi-public. So as-is users of the SDK could access this via firebase.firestore().INTERNAL.areTimestampsInSnapshotsEnabled() which we don't really want.  So I think instead I'd make this a property on Firestore:

get _areTimestampsInSnapshotsEnabled(): boolean {
  return this._config.settings.timestampsInSnapshotsEnabled;
}

Note that we use '_' for any members of our public API classes (like Firestore) that we don't want to be visible in our public API. There's a minification step as part of our build that will rename '_blah' properties to obfuscated names.

Reply:
Done, thanks for the explanation.